geometry_msgs/PoseStamped p
string arm
---
int8 result
